### ISSUE TITLE
Comments created with "Add a Comment" say "pending"

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode-pull-request-github#2822]
1. Open a PR where you haven't started a review.
2. Add a comment using the commenting widget and the "Add a Comment" button.
3. The comment will add. ✅
4. A moment later, the comment will say "Pending". ❌
![image](https://user-images.githubusercontent.com/38270282/124485464-df930600-ddac-11eb-9801-5d090664a5f3.png)

**Comment by alexr00** [comment]
There are two things going on here:
1. When we add the comment, there is a quirk of GitHub that briefly makes the comment a "pending" comment. This is just part of the API and is expected. Once the review is submitted (there is a hidden review associated with single comments, this is also just the way it works), then the comment is no longer pending. We do properly submit the review; however these leads to the second thing that's going on...
2. After the review is submitted and the comment is no longer "pending" we _should_ update the associated comment threads. We are not doing that and this is a bug.

So, in short, what should happen:
1. The comment is added and doesn't show "Pending".
2. A moment later, it does show "Pending".
3. A moment after that, "Pending" goes away again.

**Comment by negiraju** [comment]
Hey, this is test comment

**Comment by IvanZhantalay** [comment]
test

**Comment by Cuongdc89** [comment]
test

**Comment by ttmthanh** [comment]
test

**Comment by zhwuwuwu** [comment]
test

**Comment by Pony-Qinpeng** [comment]
test
